packages/mdc-ripple/_mixins.scss

```diff
@@ -140,13 +142,13 @@
 
   // Common styles for upgraded surfaces (some of these depend on custom properties set via JS or other mixins)
 
-  &.mdc-ripple-upgraded::before {
+  &.mdc-ripple-upgraded#{$ripple-target}::before {
```

Comment:
This seem like `.mdc-ripple-upgraded` and ripple-target are at the same level without the space between them. To make this explicit we can use nested block and remove space prefix from `$ripple-target`.

See this [gist from SassMeister](https://gist.github.com/abhiomkar/efc3e1131be96a6ed43632986726dce0) for example. `$ripple-target` is set to `&` when it is null to default it to root.

Reply:
Love it! Thanks for the great suggestion.